internal/configs/configurator.go

```diff
@@ -637,3 +643,13 @@ func (cnf *Configurator) GetIngressCounts() map[string]int {
 
 	return counters
 }
+
+// GetVSCounts returns the total count of Vs/Vsr resources that are handled by the Ingress Controller
+func (cnf *Configurator) GetVSCounts() (vsCount int, vsrCount int) {
```

Comment:
I suggest expanding the abbreviation for clarity and consistency with GetIngressCounts:
GetVirtualServerCounts

In the comments and other places, if you abbreviate VirtualServer and VirtualServerRoute, better to use "VS" and "VSR" instead of "Vs" and "Vsr"